### PR TITLE
Extract app name and navigation entries for l10n

### LINKED ISSUE #3292: Extract navigation.name from info.xml for translation
Follow up from #2822

### PR DESCRIPTION
**Pull request by MorrisJobke**
* fixes #3292 for the server

### CODE REVIEW

**Comment by MorrisJobke** [comment]
As this is only administrative stuff needed for the transifex sync I will merge it.

File: l10n/l10n.pl [hunk 1]
```diff
@@ -139,6 +141,7 @@ ()
 			print "    Reading $file\n";
 			`xgettext --output="$output" $joinexisting $keywords --language=$language "$file" --add-comments=TRANSLATORS --from-code=UTF-8 --package-version="8.0.0" --package-name="ownCloud Core" --msgid-bugs-address="translations\@owncloud.org"`;
```
Comment (nickvergessen) [inline l10n/l10n.pl]: this line looks odd

Comment (MorrisJobke) [inline l10n/l10n.pl]: Why? It's only metadata that is never used ;)

Comment (MorrisJobke) [inline l10n/l10n.pl]: Why? It's only metadata that is never used ;)

**Pull request merged**